src/app/system/manage-scheduler-jobs/manage-scheduler-jobs.component.ts

```diff
@@ -125,7 +125,10 @@ export class ManageSchedulerJobsComponent implements OnInit {
 
   runSelectedJobs(): void {
     this.selection.selected.forEach((job) => {
-      this.systemService.runSelectedJob(job.jobId);
+      this.systemService.runSelectedJob(job.jobId)
```

Comment:
How does it help?

Reply:
For some reason. If we don't use the suscribe the HTTP call is not executed and that was the problem.